styles.css

```diff
@@ -0,0 +1,54 @@
+#title {
+    text-align: center;
+}
+
+#survey-form {
+    text-align: center;
+    width: 40%;
+    background-color: grey;
+    margin: auto;
+    padding: 20px;
+    border-color: aqua;
+    border-radius: 10px;
+
+}
+
+p {
+    text-align: center;
+}
+
+Label {
+    color:#388080;
+    background-color:#eceff1; 
+    width: 25%;
+    padding: 4px;
+    margin: 10px;
```

Comment:
- [ ] Good Job for using GitHub inline comments feature but the suggested changes doesn't have an explanation, a "why", for each requested change.
For example : "Kindly, I recommend you to adjust margin to 5px for label element , that would improve  the page more accessibility"

Reply:
- [ ] [REQUIRED ] Please make sure that you implemented changes requested by the previous reviewer before submitting and please note that it is important to be polite and always tell the reason, **WHY** you suggest something or request a change as requested by the previous reviewer :smile: 

> * [ ]  Good Job for using GitHub inline comments feature but the suggested changes doesn't have an explanation, a "why", for each requested change.
>   For example : "Kindly, I recommend you to adjust margin to 5px for label element , that would improve  the page more accessibility"

Reply:
- [ ] [REQUIRED ] Please make sure that you implemented changes requested by the previous reviewer before submitting and please note that it is important to be polite and always tell the reason, **WHY** you suggest something or request a change as requested by the previous reviewer :smile: 

> * [ ]  Good Job for using GitHub inline comments feature but the suggested changes doesn't have an explanation, a "why", for each requested change.
>   For example: "Kindly, I recommend you to adjust margin to 5px for label element , that would improve  the page more accessibility"

Reply:
but i did as i said it would look even nicer as I was polite at the same time as I was also telling the reason why he should make a change

Reply:
Hi @shobeee , I think it will be better to phrase it like this "Kindly, I recommend you to adjust margin to 5px for label element .... because it would look even nicer as I was polite at the same time  ..."

Reply:
ok done bro i see exactly what do you mean

Reply:
- Good job implementing some of the changes suggested by the previous reviewer 👍 👍 . But there can still be some improvements to be made, your review is not clear and concise. It is important to give clear reviews so the author understands the reason why he needs to make the suggested changes. 🙏 Also, as the first reviewer suggested, a reason for requesting this change could be "...to improve the page's accessibility". I suggest you try to fix this issue 😄

Reply:
please take a look now bro